### PR TITLE
[PT] Add javascript sample to enhance tutorial 4

### PR DESCRIPTION
**Pull request by haoyunfeix**
Enhancement for pt_tutorial_4_web sample
1. Register special person by mouse click instead of automatic register
all person.
2. Registered person will show within a green box.

### CODE REVIEW

**Comment by huningxin** [comment]
@haoyunfeix , may I know why this PR include the C++ implementation?

**Comment by haoyunfeix** [comment]
The C++ parts copied from pt_tutorial_4_web/cpp/, I just want to keep the same structure with other samples. But you're right, keeping the javascript code is enough. I'll send a new PR later.

File: samples/pt_tutorial_4+_web/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+
+
+#pt_tutorial_4_web
```
Comment (huningxin) [inline samples/pt_tutorial_4+_web/README.md]: I am concerning the `+`. Could we name this sample as `pt_tutorial_4.1_web`?

Please update the `README.md` to include the new interaction of this sample.

Thanks!

**Comment by haoyunfeix** [comment]
@huningxin, PTAL

**Comment by huningxin** [comment]
lgtm. Thanks for the PR!

**Pull request merged**